### DISCUSSION TITLE
[Feature] external libraries and partner sharing

### DISCUSSION THREAD

**Discussion by breitsi** [immich-app/immich#11763] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

We use external libraries to share our (old and new) pictures within our family. Until now, each account has had to add its own library, scan all the pictures, recognize all the faces, and manage all the identified people individually. It would be nice if external libraries could be shared with more than one person, along with all the associated metadata.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#4500
